src/app/progress/Pursuits.tsx

```diff
@@ -93,16 +111,56 @@ export function PursuitsGroup({
         onSelectedFiltersChanged={setBountyFilters}
         pursuitsInfo={pursuitsInfo}
       />
+      {questPillsComponent}
       <PursuitGrid>
         {pursuits.sort(sortPursuits).map((item) => (
           <Pursuit
             item={item}
             key={item.index}
-            searchHidden={!matchBountyFilters(item, bountyFilters, pursuitsInfo)}
+            searchHidden={
+              !(
+                (activeTraitHashes.length === 0 ||
+                  activeTraitHashes.some((hash) =>
+                    defs.InventoryItem.get(item.hash).traitHashes?.includes(hash)
+                  )) &&
+                matchBountyFilters(item, bountyFilters, pursuitsInfo)
+              )
+            }
             hideDescription={hideDescriptions}
           />
         ))}
       </PursuitGrid>
     </>
   );
 }
+
+function useQuestsFilter(
+  active: boolean | undefined
+): [comp: React.ReactNode, activeTraitHashes: number[]] {
+  const defs = useD2Definitions()!;
+
+  const options = useMemo(
+    () =>
+      _.compact(
+        pursuitCategoryTraitHashes.map((hash) => {
+          const def = defs.Trait.get(hash);
+          return def && { key: hash, content: def.displayProperties.name };
+        })
+      ),
+    [defs.Trait]
+  );
+
+  const [selectedFilters, setSelectedFilters] = useState<Option<number>[]>([]);
```

Comment:
Is it problematic to have two sets of state for selected filters? For example, can you not shift-select both quest type and other bounty guide pills?

Reply:
It's not hugely problematic -- they're just two different rows of buttons -- but it's probably a bit confusing. I can try integrating it into the existing bounty guide.

Reply:
I moved it to the existing bounty guide and it's probably better that way. Also included the trait icons for some flavor.